presto-native-execution/presto_cpp/main/http/CMakeLists.txt

```diff
@@ -20,14 +20,16 @@ target_link_libraries(
   velox_memory
   velox_exception
   ${PROXYGEN_LIBRARIES}
+  fizz::fizz
+  wangle::wangle
```

Comment:
fizz and wangle were part of PROXYGEN_LIBRARIES before, but I didn't want to have them defined in the Find module, so they are now here.